Nodejs/Product/TestAdapter/TestFrameworks/Jest/jest.js

```diff
@@ -51,14 +53,15 @@ const run_tests = function (context) {
             });
         }
 
-        const config = {
+        let projectFolderJsonReady = projectFolder.replaceAll('\\', '\\\\');
+        const argv = {
             json: true,
             reporters: [[__dirname + '/jestReporter.js', { context }]],
-            testMatch: [context.testCases[0].testFile]
```

Comment:
I believe we launch a run for each file, so removing the `context.testCases[0].testFile` and instead adding all found tests will cause to run them multiple times. 

I agree running this only once will be a better approach, but some changes on NTVS will be required for that.

Reply:
Yep, just realized I was adding all of them! Changed it to the test file but inside the config variable.